ui/tests/acceptance/secrets/backend/engines-test.js

```diff
@@ -0,0 +1,38 @@
+import { currentRouteName } from '@ember/test-helpers';
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import mountSecrets from 'vault/tests/pages/settings/mount-secret-backend';
+import backendsPage from 'vault/tests/pages/secrets/backends';
+import authPage from 'vault/tests/pages/auth';
+
+module('Acceptance | engine/disable', function(hooks) {
+  setupApplicationTest(hooks);
+
+  hooks.beforeEach(function() {
+    return authPage.login();
+  });
+
+  test('disable engine', async function(assert) {
+    // first mount an engine so we can disable it.
+    let enginePath = `alicloud-${new Date().getTime()}`;
+    await mountSecrets.visit();
+    await mountSecrets.selectType('alicloud');
```

Comment:
`enable` should do the visit and selectType stuff for you so we shouldn't need lines 18 and 19 https://github.com/hashicorp/vault/blob/72ac6fa606b0fcd14c620e90576e88711dce09f9/ui/tests/pages/settings/mount-secret-backend.js#L13